x-pack/plugins/spaces/public/share_saved_objects_to_space/components/share_to_space_flyout.tsx

```diff
@@ -166,39 +204,18 @@ export const ShareSavedObjectsToSpaceFlyout = (props: Props) => {
       return <EuiLoadingSpinner />;
     }
 
-    // Step 1a: assets loaded, but no spaces are available for share.
-    // The `spaces` array includes the current space, so at minimum it will have a length of 1.
-    if (spaces.length < 2) {
-      return (
-        <EuiEmptyPrompt
-          body={
-            <p>
-              <FormattedMessage
-                id="xpack.spaces.management.shareToSpace.noSpacesBody"
-                defaultMessage="There are no eligible spaces to share into."
-              />
-            </p>
-          }
-          title={
-            <h3>
-              <FormattedMessage
-                id="xpack.spaces.management.shareToSpace.noSpacesTitle"
-                defaultMessage="No spaces available"
-              />
-            </h3>
-          }
-        />
-      );
-    }
-
-    const showShareWarning = currentNamespaces.length === 1;
+    const activeSpace = spaces.find((x) => x.isActiveSpace)!;
+    const showShareWarning =
+      spaces.length > 1 && arraysAreEqual(currentNamespaces, [activeSpace.id]);
     // Step 2: Share has not been initiated yet; User must fill out form to continue.
     return (
       <ShareToSpaceForm
+        coreStart={coreStart!}
```

Comment:
Can be removed since it's unused by child

Reply:
[It's actually used](https://github.com/elastic/kibana/pull/76132#discussion_r498483259)